runtime/util_core/j9argscan.c

```diff
@@ -404,6 +502,3 @@ void scan_failed_unsupported(J9PortLibrary * portLibrary, char* module, char *sc
 	PORT_ACCESS_FROM_PORT(portLibrary);
 	j9tty_printf(PORTLIB, "<%s: system configuration does not support option --> '%s'>\n", module, scan_start);
 }
-
```

Comment:
please leave one empty line at the end of file

Reply:
please double check that <EOF> does not stay at the same line as a `}`. It might be a reason for fantom compilation errors on some compilers